owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/console/maintenance/task/component/TaskExecutionResultPanel.java

```diff
@@ -73,11 +76,17 @@ protected void onConfigure() {
 					}
 				});
 				
-				contexteItemsItem.add(new SerializedItemListView<BatchReportItem>("itemListView", itemListModel) {
+				contexteItemsItem.add(new CollectionView<BatchReportItem>(
```

Comment:
I think you missed the second constructor of `CollectionView`: `CollectionView(String, IModel<? extends Collection<T>>, Function<? super T, ? extends IModel<T>>)`. It's a bit simpler than using `ReadOnlyCollectionModel`.
Could you please check the places you used `CollectionView` and fix that? I wouldn't ask for this in another project, be here, the code of OWSI-Core also serves as an example, so we'd better avoid bad usage patterns.